04week/loop.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const  carsInReverse = ['honda', 'lexus' ,'chevy', 'gmc', 'cadillac'];
+
+const carsInArray = (carsInReverse) => {
+  for(let i = carsInReverse.length - 1; i >= 0 ; i--){
+    console.log(carsInReverse[i]);
+  }
+}
+carsInArray(carsInReverse);
+////////////////////////////////////////////////////////////////////////////////
+
+const person = {
+  firstName: "Jane",
+  lastName: "Doe",
+  birthDate: "Jan 5, 1925",
+  gender: "female"
+}
+for(let property in person){
+  if(property == 'birthDate'){
+    console.log(property + " " + person[property])
+  }else{
+    console.log(property);
+  }
+}
+////////////////////////////////////////////////////////////////////////////////
+
+let i = 1;
+while(i <= 1000){
+//  console.log(i);
+  i++;
+}
+////////////////////////////////////////////////////////////////////////////////
+let o = 1;
+ do{
+  // console.log(o);
+   o++;
+ }
+ while(o <= 1000);
+
+
+// When is a for loop better than a while loop?
+// when you need to check check the condition before the loop starts
```

Comment:
both for loop and while loop check the condition in the same place. What is a big risk in a while loop?